resources/css/common.css

```diff
@@ -780,11 +435,6 @@ li p:last-child,
   opacity: 0.7;
 }
 
-.svg-shadow {
-  -webkit-filter: drop-shadow(1px 1px 2px rgba(0, 0, 0, 0.7));
-  filter: drop-shadow(1px 1px 2px rgba(0, 0, 0, 0.5));
-}
```

Comment:
The icon shadow looks weird on light themes.

![Screenshot from 2022-09-26 13-24-16](https://user-images.githubusercontent.com/10744960/192253854-b2636826-66c0-416b-9eed-6942f5378acc.png)